pkg/wrangler/context.go

```diff
@@ -0,0 +1,343 @@
+/*
+Package wrangler contains functions for creating a management context with wrangler controllers.
+*/
+package wrangler
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+
+	"github.com/rancher/lasso/pkg/controller"
+	"github.com/rancher/lasso/pkg/dynamic"
+	"github.com/rancher/norman/types"
+	managementv3api "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/pkg/settings"
```

Comment:
we have our own pkg/settings

Reply:
Do we have the necessary functions in that pkg/settings?   Are you referring to extensions/settings?

Reply:
Clarified offline